manifests/gcp_marketplace/schema.yaml

```diff
@@ -101,33 +101,59 @@ properties:
     x-google-marketplace:
       type: NAMESPACE
   serviceAccountCredential:
-    title: Service Account credentials used to call other GCP services.
+    title: |-
+      Service Account credentials used to call other GCP services.
     description: |-
-      To be able to call other GCP services, we need to be
-      authenticated. This field is used to store the content of the service account
+      To be able to call other GCP services from the pipeline, we need to be authenticated.
+      This field is used to store the content of the service account
       JSON file. It can be encoded using base64 instead of messing with JSON format.
+      To get the encoded credential, run
+      $ gcloud iam service-accounts keys create application_default_credentials.json --iam-account [your-service-account] && cat application_default_credentials.json | base64
```

Comment:
not sure whether the special characters here '$', '&' will be escaped by MKP UI, we may need to double check it manually after another staging in MKP

Reply:
i verified it and should be ok.